src/components/schedule-page/schedule-page.component.tsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import { TableComponent } from "./table/table.component";
 import { ToolbarComponent } from "./toolbar.component";
+import Header from "../common-components/header/header";
 
 export default function SchedulePage(): JSX.Element {
   return (
     <React.Fragment>
       <div className="header">
+        <Header />
```

Comment:
Próbowałeś dodać ten komponent w `app.tsx`? 
Odrazu na `ReactRouter`. 
Mi się wydaje, że wtedy to też będzie działać, ale będzie miało taką zaletę, że nagłówek będzie odrazu dodany do wszystkich stron